docs/howto/features/buckets.md

```diff
@@ -310,19 +320,60 @@ On AWS, we would need to set up [cross account S3 access](https://aws.amazon.com
    }
    ```
 
-   You can add additional permissions to the bucket if needed here.
+   here,
+   1. `Statement.Principal.AWS` is the ARN determined from step 1. This is the only item that needs
+      to be supplied by the 2i2c engineer.
+   2. `Statement.Action` is the list of S3 permissions to be granted. The default allows readonly
+      access, with the ability to list objects and get their data. The community champion is welcome
+      to modify this to fit their needs.
+   3. `Statement.Resource` is the list of buckets this policy is applied to. There are two entries for
+      each bucket, as some permissions apply to the bucket itself and some apply to the contents of
+      the bucket. We don't need to know the list of buckets here, the community champion can supply this.
+
+3. **Who**: 2i2c Engineering Team
+
+   Communicate to the community champion that they need to [attach a bucket policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/add-bucket-policy.html)
+   to the S3 buckets. You can use the following template.
 
    ```{note}
-   You can list as many buckets as you want, but each bucket needs two entries -
-   one with the `/*` and one without so both listing the bucket as well as fetching
-   data from it can work
+   To start the process, we need to attach policies to the appropriate S3 bucket to grant access
+   to the hub. I've generated these templates:
+
+   {{ JSON from step 2. Include multiple ones if we are doing this for multiple hubs }}
+
+   {{ if a staging hub is included }}
+   This includes access permissions for staging hub we use to test out changes to the main hub,
+   so those two can be in sync.
+   {{ end if }}
+
+   You need to fill in the list of S3 buckets you want granted access under the "Resource" section.
+   Note that there are two entries for each bucket.
+
+   This policy, as written, will grant readonly access to the buckets from the hub. Users
+   on the hub will be able to list objects and retrieve them. If you'd like to grant other
+   kinds of access, adjust the list under 'Actions' accordingly.
+
+   AWS has [some documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/add-bucket-policy.html)
+   on how to attach this bucket policy. Once attached, can you give us the full content of the
+   policy you attached? That would help us take next steps in granting access to these buckets.
```

Comment:
Since its only possible to have one bucket policy (but multiple statements in it), the bucket admins are required to merge any existing policy with the policy we provide - by appending statements and avoiding statement duplicates.

The text needs to be updated to reflect this.